scripts/reinstall-rapiddisk-c.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# For Debian: v1.01
+# Warning: ECC Memory recommended.
+# Reinstall Part 3/3 for Rapiddisk 8.2.0 Linux.
```

Comment:
I want to make this somewhat generic. If we can instead use "RapidDisk versions 8 and above"